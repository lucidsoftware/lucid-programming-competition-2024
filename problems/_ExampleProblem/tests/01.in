version https://git-lfs.github.com/spec/v1
oid sha256:dbba2749014ee8a4354b495087ddae9a063d7f9bea272fb31611ec5cd948c809
size 149
