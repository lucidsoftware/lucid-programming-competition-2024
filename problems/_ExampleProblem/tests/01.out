1 allyallosaurus igoriguanodon
2 n/a
3 pennypterodactyl
4 n/a
5 rickyraptor
